playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup/iam.tf

```diff
@@ -36,7 +36,8 @@ resource "google_project_iam_member" "cloud_build_roles" {
     "roles/iam.serviceAccountUser",
     "roles/datastore.indexAdmin",
     "roles/storage.admin",
-    "roles/logging.logWriter"
+    "roles/logging.logWriter",
+    "roles/datastore.owner"
```

Comment:
Do we really need owner?

Reply:
@rshamunov I've tested without datastore IAM role, with owner and user
Without it would give error (PERMISSION_DENIED) when deploying
Owner/User worked fine. Leaving User